blobstore/blobstore.go

```diff
@@ -106,6 +111,41 @@ func handler(r *data.FileRepo) http.Handler {
 func main() {
 	defer shutdown.Exit()
 
+	usage := `
+usage: flynn-blobstore <command> [<args>...]
+
+Commands:
+	help        show usage for a specific command
+	cleanup     delete file blobs from default backend 
+	migrate     move file blobs from default backend to a different backend
```

Comment:
Ok well the usage requires a `<command>`, so you would need to make it optional, but personally I think you should just add `server` as a command which is documented here.